README.md

```diff
@@ -0,0 +1,200 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+- Installation of NPM
+- Installation of Live Server plugin in Visual Studio Code
+
+<!--
+Example command:
+```sh
+ gem install rails
+```
+ -->
+### Setup
+
+Go to github and find the repository with code-review
+Click on code and copy then go to your gitbash cli on your computer Clone this repository to your desired folder
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:KUBAHO3/Hello-Microverse.git
+```
+--->
+### Install
+Install this project with:
+npm
+Example command:
+```sh
+  cd my-projectFolder
+  npm install
+```
+### Usage
+To run the project, execute the following command:
+Open the cloned folder in visual Studio. Right click the index.html source code and click 
+  'Open with Live Server'. The project will open automatically in your default browser.
+
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+### Deployment
+You can deploy this project using:
+GitHub Pages
+Example:
+```sh
+git@github.com:rosemutai/My-Portfolio.git
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **KUBAHO LINNE Heaven**
+- GitHub: [rosemutai](https://github.com/rosemutai)
+- Twitter: [__rosemutai](https://twitter.com/__rosemutai)
+- LinkedIn: [Rose Mutai](https://www.linkedin.com/in/rosemutai/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+### Key Features <a name="key-features"></a>
+- [ ] **making the porfolio responsive**<br/>
+- [ ] **Adding Animation**<br/>
+- [ ] **Making it dynamic**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!<br/>
+Feel free to check the [https://github.com/rosemutai/My-Portfolio/issues.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/rosemutai/My-Portfolio/blob/main/LICENSE) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE - https://choosealicense.com/licenses/mit/